packages/cli/src/lib/container/docker_helper.ts

```diff
@@ -60,3 +65,63 @@ export const pullImage = function (resource: string) {
   const args = ['pull', resource]
   return cmd('docker', args)
 }
+
+export const getDockerfiles = function (rootdir: string, recursive: boolean) {
+  const match = recursive ? './**/Dockerfile?(.)*' : 'Dockerfile*'
+  let dockerfiles: string[] = globSync(match, {
+    cwd: rootdir,
+    nodir: true,
+  })
+  if (recursive) {
+    dockerfiles = dockerfiles.filter((df: string) => df.match(/Dockerfile\.[\w]+$/))
+  } else {
+    dockerfiles = dockerfiles.filter((df: string) => df.match(/Dockerfile$/))
+  }
+
+  return dockerfiles.map((file: string) => path.join(rootdir, file))
+}
+
+export const getJobs = function (resourceRoot: string, dockerfiles: string[]) {
+  return dockerfiles
+  // convert all Dockerfiles into job Objects
+    .map(dockerfile => {
+      const match = dockerfile.match(DOCKERFILE_REGEX)
+      if (!match) throw new Error(`Invalid Dockerfile: ${dockerfile}`)
+      const proc: string = (match[1] || '.standard').slice(1)
+      const dockerfileJobs: Record<string, any> = {
```

Comment:
Is the use of `any` necessary here? It seems like this object is structured and can be typed.

Reply:
@justinwilaby I had to convert the same lib files and I've added structured types here. Maybe we can approve as is and then I refactor slightly when merging changes from my PR?

Reply:
Sounds good @sbosio!